agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/logging/OperationLogger.java

```diff
@@ -63,7 +63,13 @@ public void recordFailure(String failureMessage, AzureMonitorMsgId msgId) {
   }
 
   // failureMessage should have low cardinality
-  public void recordFailure(String failureMessage, @Nullable Throwable exception) {
+  public void recordFailure(String failureMessage) {
+    if (aggregatingLogger != null) {
+      aggregatingLogger.recordWarning(failureMessage, null);
```

Comment:
```suggestion
      aggregatingLogger.recordFailure(failureMessage, null);
```

Reply:
there is no recordFailure(?)

Reply:
never mind.. it's in aggregatingLogger. Sorry, thought you overloaded the method below, line 72 for this.